documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    isinstance(obj, list)       |    instance?(obj, <list>)                     |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    [x y z]                     |    list(x, y, z)                              |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    len ls                      |     ls.size                                   |
+|                                |     size(ls)                                  |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    ls1 + ls2 + ls3             |    concatenate(ls1, ls2, ls3)                 |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    ls[::-1]                    |    reverse(ls)                                |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    ls[n]                       |    ls[n]                                      |
+|                                |    element(ls, n)                             |
++--------------------------------+-----------------------------------------------+
+| .. code-block:: python         | .. code-block:: dylan                         |
+|                                |                                               |
+|    obj in ls                   |    member?(obj, ls)                           |
++--------------------------------+-----------------------------------------------+
+| N/A                            | .. code-block:: dylan                         |
+|                                |                                               |
+|                                |    member?(obj, ls, test: \==)                |
++--------------------------------+-----------------------------------------------+
+
+Symbols
+^^^^^^^
```

Comment:
Similar here.  Python folks won't be looking for a way to use symbols and `str` isn't really the same thing.  Possibly if you switch to using `sys.intern` instead it will work.

Reply:
`sys.intern` 
```
>>> a = sys.intern('a')
>>> b = sys.intern('A')
>>> a is b
False
``

Reply:
That's what I would expect.